dev/top_printers.ml

```diff
@@ -64,8 +64,10 @@ let ppwf_paths x = pp (Rtree.pp_tree prrecarg x)
 let envpp pp = let sigma,env = Pfedit.get_current_context () in pp env sigma
 let rawdebug = ref false
 let ppevar evk = pp (Evar.print evk)
-let ppconstr x = pp (Termops.print_constr (EConstr.of_constr x))
-let ppeconstr x = pp (Termops.print_constr x)
+let pr_constr = Printer.pr_constr [@ocaml.warning "-3"]
+let pr_econstr = Printer.pr_econstr [@ocaml.warning "-3"]
```

Comment:
Why not fix the warning instead of deactivating it?

Reply:
This is addressed in a different PR so I was lazy here.

Reply:
Namely such printers are removed.

Reply:
After a second thought let me indeed address this here. I have so many PRs related to this that I am not sure where I am.